### ISSUE TITLE
[Logging Issue] Incorrect flag in logs after resetting cluster using a restore-path

### ISSUE THREAD

**Issue by rancher-max** [k3s-io/k3s#2244]
**Environmental Info:**
K3s Version: 

master branch, commit id: `beab211685805a481ffa61070ec53ddefd1adf03`

**Describe the bug:**

After restarting the cluster using `--cluster-reset` and `--cluster-reset-restore-path` the logs show:
```
Sep 14 16:38:32 ip-172-31-30-163 k3s[6621]: time="2020-09-14T16:38:32.289241797Z" level=info msg="etcd already restored from a snapshot. Restart without --snapshot-restore-path flag. Backup and delete ${datadir}/server/db on each peer etcd server and rejoin the nodes"
```
It should show `--cluster-reset-restore-path` instead of `--snapshot-restore-path` since we changed the name of the flag during development.

**Steps To Reproduce:**

- Reset a k3s cluster using --cluster-reset and --cluster-reset-restore-path

**Comment by MonzElmasry** [comment]
Resolved via :  https://github.com/k3s-io/k3s/pull/2307

**Comment by ShylajaDevadiga** [comment]
Log message is fixed in k3s version v1.21.1-rc1+k3s1 as seen below when --cluster-reset and --cluster-reset-restore-path is passed
```
INFO[2021-05-19T06:06:07.362740449Z] k3s is up and running                        
INFO[2021-05-19T06:06:10.135600673Z] Etcd is running, restart without --cluster-reset flag now. Backup and delete ${datadir}/server/db on each peer etcd server and rejoin the nodes 
```
